src/main/java/Display.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Display {
+    public void printGreeting() {
+        System.out.println("Welcome to TicTacToe!");
+    }
+
+    public void promptPlayer(String playerMark) {
+        System.out.println(String.format("Make a move: %s", playerMark));
+    }
+
+    private List currentGridCells(Board board, int size) {
+        List currentGridAsArray = new ArrayList<>();
+        Board boardClass = board;
+        int gridSize = size;
+        int count = 1;
```

Comment:
I think that this is a subtle bit of duplication going back to the Board class. Your display needs to know that it should start at 1 and not 0. Maybe that's okay, but something to think about.

Reply:
✅ moved to `Board` and updated method to run from 0 PR #6